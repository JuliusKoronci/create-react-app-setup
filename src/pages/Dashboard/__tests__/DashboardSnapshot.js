import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Dashboard from '../Dashboard';

const wrapper = shallow(<Dashboard />);
describe('(Component) Dashboard', () => {
	it('Should match snapshot', () => {
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});

