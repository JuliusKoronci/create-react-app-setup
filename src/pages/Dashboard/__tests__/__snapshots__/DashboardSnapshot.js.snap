// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) Dashboard Should match snapshot 1`] = `
<div>
  Dashboard
</div>
`;
